client/src/pages/Admins/AdminFinance/AddTransactionDialog/AddTransactionDialog.js

```diff
@@ -34,9 +36,22 @@ function AddTransactionDialog (props) {
         setErrorTokenAmount(validationResult);
         setAmountValid(validationResult === '');
     };
-
+    const handleDateChange = (event) => {
+        const input = event.target;
+        setDueDate(input.value);
+        const yearMonthDayString = new Date(Date.now()).toISOString().slice(0, 10);
+        const validationResult = new Date(input.value) < new Date(yearMonthDayString) ? 'WRONGDATE' : '';
```

Comment:
Does this comparison work even after you've sliced a date string? I'm not sure, just asking

Reply:
Yes, it works:)

Reply:
Please check the new Date Picker on Finance Admin 'Add new debit' and Personal page birthday. Does it work well on Safari?